bluemira/plasma_physics/collisions.py

```diff
@@ -132,6 +204,22 @@ def coulomb_logarithm(temperature: float, density: float) -> float:
     Returns
     -------
     Coulomb logarithm value
+
+    Notes
+    -----
+    The Coulomb logarithm is calculated using the formula:
+
+    .. math::
+
+        \\ln{\\Lambda} = \\ln{\\left(1 + \\left(\\frac{\\lambda_{Debye}}
+        {b_{min}}\\right)^2\\right)^{1/2}}
+
+    where:
+
+    - :math:`\\lambda_{Debye}` is the Debye length,
+
+    - :math:`b_{min}` is the minimum impact parameter
```

Comment:
could you include `\max(b_{90},\lambda_{th})` for what b_min is in the definitions

Reply:
done